src/csharp/Grpc.Core/Grpc.Core.csproj

```diff
@@ -46,8 +46,12 @@
       <PackagePath>runtimes/win/native/grpc_csharp_ext.x86.dll</PackagePath>
       <Pack>true</Pack>
     </Content>
-    <Content Include="Grpc.Core.targets">
-      <PackagePath>build/net45/</PackagePath>
+    <Content Include="..\nativelibs\csharp_ext_monoandroid81_x86\libgrpc_csharp_ext.so">
+      <PackagePath>runtimes/monoandroid/armeabi-v7a/libgrpc_csharp_ext.so</PackagePath>
+      <Pack>true</Pack>
+    </Content>
+    <Content Include="..\nativelibs\csharp_ext_monoandroid81_x64\libgrpc_csharp_ext.so">
```

Comment:
nit: why `csharp_ext_monoandroid81_x64`?  This should be set to the path where the artifacts are placed by the build_artifacts CI job. We should update once we have building the artifacts for android setup.

Btw, is there anything Xamarin-specific about these artifacts (the name is "monoandroid81"?) it seems like they should work equally well on Unity (which is another platform we want to support).

Reply:
To be honest, I don't know. If I remember correctly, in `CMakeLists.txt` wasn't there a `-llog` and `-landroid` to produce the `libgrpc_csharp_ext.so` and those libraries stem from inside the Android NDK?